diff --git a/config.def.h b/config.def.h
index 0e01717..fbbb2a4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -119,6 +119,10 @@ static const char *colorname[] = {
 unsigned int defaultbg = 0;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
+
+/* Colors used for selection */
+unsigned int selectionbg = 257;
+unsigned int selectionfg = 7;
 
 /*
  * Default shape of cursor
diff --git a/x.c b/x.c
index 0422421..a001f2e 100644
--- a/x.c
+++ b/x.c
@@ -1336,9 +1336,7 @@ xdrawglyphfontspecs(const XftGlyphFontSpec *specs, Glyph base, int len, int x, i
 	}
 
 	if (base.mode & ATTR_REVERSE) {
-		temp = fg;
-		fg = bg;
-		bg = temp;
+		bg = &dc.col[selectionbg];
 	}
 
 	if (base.mode & ATTR_BLINK && win.mode & MODE_BLINK)
-- 
2.23.0

